modules/3d-tiles/src/tz3-loader.ts

```diff
@@ -0,0 +1,41 @@
+import {LoaderOptions, LoaderWithParser} from '@loaders.gl/loader-utils';
+import {DataViewFile} from '@loaders.gl/zip';
+import {parse3tz as parse3tzFromProvider} from './tz3/tz3-parser';
+
+// __VERSION__ is injected by babel-plugin-version-inline
+// @ts-ignore TS2304: Cannot find name '__VERSION__'.
+const VERSION = typeof __VERSION__ !== 'undefined' ? __VERSION__ : 'latest';
+
+/** options to load data from 3tz */
+export type TZ3LoaderOptions = LoaderOptions & {
+  tz3?: {
+    /** path inside the 3tz archive */
+    path?: string;
+  };
+};
+
+/**
+ * Loader for 3tz packages
+ */
+export const TZ3Loader: LoaderWithParser<Buffer, never, TZ3LoaderOptions> = {
+  name: '3tz',
+  id: '3tz',
+  module: 'i3s',
+  version: VERSION,
+  mimeTypes: ['application/octet-stream'],
```

Comment:
I wonder if e.g. Cesium has some vendor specific mime type for this? Worth Googling a bit?